src/librustc_incremental/persist/load.rs

```diff
@@ -117,7 +117,7 @@ pub fn load_dep_graph(sess: &Session) ->
     if sess.opts.incremental.is_none() {
         // No incremental compilation.
         return MaybeAsync::Sync(LoadResult::Ok {
-            data: (PreviousDepGraph::new(SerializedDepGraph::new()), FxHashMap())
+            data: (PreviousDepGraph::new(SerializedDepGraph::new()), FxHashMap::default())
```

Comment:
`<(_ , _)>::default()` :laughing:

Reply:
`PreviousDepGraph` wraps a `SerializedDepGraph` so that's probably not the best idea.